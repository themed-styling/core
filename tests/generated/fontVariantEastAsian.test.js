import 'jest-styled-components'

import { fontVariantEastAsian } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(fontVariantEastAsian, 'fontVariantEastAsian')
