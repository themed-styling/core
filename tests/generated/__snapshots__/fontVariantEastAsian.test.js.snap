// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fontVariantEastAsian renders 0 1`] = `
Array [
  .c0 {
  font-variant-east-asian: 0;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: 0;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: 0;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders function 1`] = `
Array [
  .c0 {
  font-variant-east-asian: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: has;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: has;
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders invalid theme path 1`] = `
Array [
  .c0 {
  font-variant-east-asian: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders negative number 1`] = `
Array [
  .c0 {
  font-variant-east-asian: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    font-variant-east-asian: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    font-variant-east-asian: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    font-variant-east-asian: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    font-variant-east-asian: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    font-variant-east-asian: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    font-variant-east-asian: July;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: August;
  }
}

@media January {
  .c0 {
    font-variant-east-asian: September;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: November;
  }
}

@media June {
  .c0 {
    font-variant-east-asian: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    font-variant-east-asian: July;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: August;
  }
}

@media January {
  .c0 {
    font-variant-east-asian: September;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: November;
  }
}

@media June {
  .c0 {
    font-variant-east-asian: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    font-variant-east-asian: January;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: December;
  }
}

@media January {
  .c0 {
    font-variant-east-asian: April;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: May;
  }
}

@media June {
  .c0 {
    font-variant-east-asian: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    font-variant-east-asian: January;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: December;
  }
}

@media January {
  .c0 {
    font-variant-east-asian: April;
  }
}

@media April {
  .c0 {
    font-variant-east-asian: May;
  }
}

@media June {
  .c0 {
    font-variant-east-asian: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders postitive number 1`] = `
Array [
  .c0 {
  font-variant-east-asian: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders string 1`] = `
Array [
  .c0 {
  font-variant-east-asian: July;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: July;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: June;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: June;
}

<div
    className="c0"
  />,
]
`;

exports[`fontVariantEastAsian renders valid theme path 1`] = `
Array [
  .c0 {
  font-variant-east-asian: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  font-variant-east-asian: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
